src/v2/USCC.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.26;
```

Comment:
```suggestion
pragma solidity ^0.8.28;
```

Reply:
Addressed in
- https://github.com/superstateinc/ustb/pull/56/commits/4e52d672532d51249bce9677ae1801a91404eda8
- https://github.com/superstateinc/ustb/pull/56/commits/863dda0005df872dec9136d026df07f918dbb2e5